package.json

```diff
@@ -21,17 +22,17 @@
   },
   "dependencies": {
     "@babel/core": "^7.10.2",
-    "@hapi/joi": "15.1.1",
     "@elastic/eui": "^26.0.0",
-    "@types/reach__router": "^1.3.5",
+    "@hapi/joi": "15.1.1",
     "@testing-library/jest-dom": "^5.10.1",
     "@testing-library/react": "^10.2.1",
     "@types/jest": "^26.0.0",
     "@types/node": "^14.0.13",
+    "@types/reach__router": "^1.3.5",
     "@types/react": "^16.9.36",
     "@types/react-dom": "^16.9.8",
     "@types/react-test-renderer": "^16.9.2",
-    "images-to-pdf": "^1.0.3",
+    "gensync": "^1.0.0-beta.1",
```

Comment:
Ensure we don't get any breaking changes and use this on production.

Reply:
this dependency was supposed to be removed. will remove